docs/v2/getting-started.md

```diff
@@ -7,12 +7,12 @@ title: Getting Started
 Alle gemeenten en andere organisaties met een autorisatiebesluit conform artikel 38 van het Besluit BRP kunnen zich aanmelden voor deelname aan de pilot BRP.
 
 1. Bekijk de [functionaliteit en specificaties](#functionaliteit-en-specificaties)
-2. Probeer en test de API [lokaal](#probeer-en-test-de-api-lokaal) of in de [proef omgeving](#probeer-en-test-de-api-in-de-proef-omgeving)
+2. Probeer en test de BRP API [lokaal](#probeer-en-test-de-api-lokaal) of in de [proef omgeving](#probeer-en-test-de-api-in-de-proef-omgeving)
 3. [Download]({{ site.onboardingUrl }}){:target="_blank" rel="noopener"} en lees het onboardingproces
 
 ## Functionaliteit en specificaties
 
-De '{{ site.apiname }}' Web API is gespecificeerd met behulp van de [OpenAPI Specification v3.0.3](https://spec.openapis.org/oas/v3.0.3).
+De '{{ site.apiname }}' BRP API Personen is gespecificeerd met behulp van de [OpenAPI Specification v3.0.3](https://spec.openapis.org/oas/v3.0.3).
```

Comment:
De '{{ site.apiname }}' variabele heeft als waarde BRP Personen. Dat betekent dat de tekst in Pages dan wordt

```
De 'BRP Personen' BRP API Personen is ....
```

Is dit het gewenste tekst?

Reply:
Heb nu de  {{ site.apiname }} veranderd in BRP API Personen, en alle andere vermeldingen verwijderd.